### DISCUSSION TITLE
Max retry

### DISCUSSION THREAD

**Discussion by ryuphi** [redis/rueidis#476] [Q&A]
I understand that there is a Retry mechanism for failures in the net and that we can disable it by means of the `DisableRetry` option. But is there a maximum retries? Or it enters in a loop until the connection is achieved in `read-only` operations?

**Comment by rueian** [comment]
Hi @ryuphi, a read-only command can be retried indefinitely until the `ctx` is canceled or its deadline is exceeded.
